### ISSUE TITLE
GoPls won't start to incomaptible platform.

### ISSUE THREAD

**Issue by emilpriver** [lspcontainers/dockerfiles#86]
Hey 👏

I get `Client 2 quit with exit code 1 and signal 0` when using GOPLS lsp. Did LSP log and go this: 
![Skärmavbild 2022-10-12 kl  00 21 54](https://user-images.githubusercontent.com/34039674/195209179-2fc36d84-72ae-4fc2-87e0-bcfb9750d1a5.png)

Also get a error with rust analyzer: 
![Skärmavbild 2022-10-12 kl  00 32 09](https://user-images.githubusercontent.com/34039674/195210405-d8a81221-3251-4c59-bcab-625b2b9870d2.png)

I think it has something to do with my machine. I have Macbook Pro M1.

I've tried to fix the problem myself but it's out of my knowledge. Let me know if I can provide with anything.

Any idea what might be wrong? :D

**Comment by emilpriver** [comment]
@erikreinert You wrote in the old issue and asked if it helped. But I get the same message with missmatched platform.

Could `--platform=linux/amd64` work when running buildx build command? @erikreinert

**Comment by emilpriver** [comment]
More information: this is due to Mac M1. I am now using Colima and it seems to be working.

**Comment by erikreinert** [comment]
Yes awesome! I was suspecting this may have been related to the M1 and can be an issue when on this architecture. I'll create a ticket as a follow up to see how we could provide images for multiple architectures - we just haven't run across this yet.

Thank you for reporting it!
